tutorials/how_to_dataloader.ipynb

```diff
@@ -0,0 +1,296 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[['Mo', 'triphenylsiloxy', 'selenido'],\n",
+       " ['W', 'methyl', 'imido4nitrophenyl'],\n",
+       " ['W', 'tertbutoxy3F', 'imidotrityl'],\n",
+       " ['Mo', 'pentafluorophenoxy', 'oxo'],\n",
+       " ['W', 'phenoxy', 'imido4methylphenyl'],\n",
+       " ['W', 'tertbutoxy3F', 'imido4hydroxyphenyl']]"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data_loader.categorical_target_decoder(y_test)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['Mo',\n",
+       " 'triphenylsiloxy',\n",
+       " 'selenido',\n",
+       " 'W',\n",
+       " 'methyl',\n",
+       " 'imido4nitrophenyl',\n",
+       " 'W',\n",
+       " 'tertbutoxy3F',\n",
+       " 'imidotrityl',\n",
+       " 'Mo',\n",
+       " 'pentafluorophenoxy',\n",
+       " 'oxo',\n",
+       " 'W',\n",
+       " 'phenoxy',\n",
+       " 'imido4methylphenyl',\n",
+       " 'W',\n",
+       " 'tertbutoxy3F',\n",
+       " 'imido4hydroxyphenyl']"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data_loader.confusion_matrix_data_adapter_categorical(y_test)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Confusion matrix with the dataloader\n",
+    "Here just an example how nice the sklearn funcitons seem to work with labeld data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([[2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],\n",
+       "       [0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],\n",
+       "       [0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],\n",
+       "       [0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0],\n",
+       "       [0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0],\n",
+       "       [0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0],\n",
+       "       [0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0],\n",
+       "       [0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0],\n",
+       "       [0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0],\n",
+       "       [0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0],\n",
+       "       [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0],\n",
+       "       [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2, 0],\n",
+       "       [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1]])"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from sklearn.metrics import confusion_matrix\n",
+    "y_true_cm = y_pred_cm = data_loader.confusion_matrix_data_adapter_categorical(y_test)\n",
```

Comment:
Nice! Suggestion to expand this: For our final submission, we could also show how the confusion matrix changes for i) a "dummy model" that only chooses the most often occuring ligand and ii) a "simple" Random Forest model that we use in our workflow, but without hyperparameter optimization (that perhaps shows a better performance than the dummy model)